Doc edits - Fundamentals (#495)
zama-ai/fhevm: docs/fundamentals/decrypt.md

# Decrypt and reencrypt

> This document explains how to decrypt and reencrypt data on the blockchain using a Key Management Service (KMS) and the Gateway service. The process ensures secure handling of sensitive data without exposing the blockchain's private key.

Reviewer: ```suggestion
This document aims to explain how the decryption and reencryption process works using the Key Management Service (KMS) and the Gateway.
```
